### PR TITLE
Feature/windows compat

### PR DESCRIPTION
**Pull request by ettiennegous**
Change default port to 1234
Add path normalise command for cross platform support
Updates to serve-debug to support windows
Update Todo's on the readme and link to reading material

### CODE REVIEW

**Review by woohsian — COMMENTED** [review]
<Ignore>

File: tools/loadComponents.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ const loadInComponentFile = async (innerDirectory, fileInner) => {
         if (path.extname(fileshort) === ".js") {
             console.log("'%s' is a component to be processed.", filename);
             const nameOfFile = path.basename(fileshort, ".js");
-            let resource = require(path.normalize('../' + fileshort));
+            let resource = require(path.normalize('../' + innerDirectory + fileshort));
```
Comment (woohsian) [inline tools/loadComponents.js]: I had to reinstate this to get it to work on my machine - can you see if it still works on yours?

File: index.js [hunk 2]
```diff
@@ -44,7 +44,12 @@ const { asyncForEach } = require('./tools/helpers'),
         let resourceFile = file.replace('.htl', '.js');
         let resourceFileFullPath = srcSpecFolder + resourceFile;
         // The reason to delete is to allow hot reloading to also rebuild components
-        delete require.cache[require.resolve('./' + resourceFileFullPath)];
+
+        // This line causes webpack to run into memory heap issues
+        // delete require.cache[require.resolve('./' + resourceFileFullPath)];
```
Comment (woohsian) [inline index.js]: Unfortunately, this is the line that is causing webpack to crash - I think webpack is getting confused where to look and starts loading unnecessary modules

File: index.js [hunk 3]
```diff
@@ -33,25 +33,25 @@ const { asyncForEach } = require('./tools/helpers'),
...
-          fileshort = filename.substring(filename.lastIndexOf('/') + 1),
-          fileshorthtml = fileshort.replace('.htl', '.html'),
-          resourceFile = fileshorthtml.replace('.html', '.js');
-
+        
+        let filename = path.join(__dirname, srcHTLFolder, file);
+        // Previous implementation from Node CLI
+        // const filename = process.argv[2];
+        // New implementation which loops through the test folder
+        //let fileshort = filename.substring(filename.lastIndexOf('/') + 1);
```
Comment (arollason) [inline index.js]: Could we remove this line instead of commenting it out?

File: index.js [hunk 4]
```diff
@@ -33,25 +33,25 @@ const { asyncForEach } = require('./tools/helpers'),
...
+        let fileshorthtml = file.replace('.htl', '.html');
+        let resourceFile = file.replace('.htl', '.js');
+        let resourceFileFullPath = srcSpecFolder + resourceFile;
         // The reason to delete is to allow hot reloading to also rebuild components
-        delete require.cache[require.resolve(srcSpecFolder + resourceFile)];
-        let resource = await require(srcSpecFolder + resourceFile);
 
+        // This line causes webpack to run into memory heap issues
+        // delete require.cache[require.resolve('./' + resourceFileFullPath)];
```
Comment (arollason) [inline index.js]: same here :)

File: webpack.config.js [hunk 5]
```diff
@@ -90,12 +91,12 @@ const config = {
                 presets: ['@babel/preset-env']
               }
             },
-            {
-              loader: 'eslint-loader',
-              options: {
-                fix: true
-              }
-            }
+            // {
```
Comment (arollason) [inline webpack.config.js]: Why are we removing eslint?

Comment (ettiennegous) [reply webpack.config.js]: @woohsian is this a temporary removal?

Comment (woohsian) [reply webpack.config.js]: @arollason @ettiennegous Yes it is temporary

**Comment by woohsian** [comment]
@ettiennegous @arollason I have removed some commenting and made some further fixes, ready to merge if approved

**Review by arollason — APPROVED** [review]

**Pull request merged**